definitions/npm/nano-md5_v1.x.x/test_nano-md5_v1.x.x.js

```diff
@@ -0,0 +1,8 @@
+import md5 from "nano-md5";
+
+// $ExpectError: accept only strings
+md5(6);
```

Comment:
Test positive version that default export works.

Reply:
Positive test was added.